packages/annotation-converter/src/writeback.ts

```diff
@@ -43,6 +43,26 @@ function revertObjectToRawType(references: Reference[], value: any) {
                 String: value.valueOf()
             };
         }
+    } else if (value.isInt?.()) {
```

Comment:
While i do know where the isDecimal is coming from, where do those isString / isInt / isData ... come from ?
Otherwise we would need to do similar code than in revertValueToRawType (and potentially centralize it)

Reply:
It is coming from here. https://github.wdf.sap.corp/ux-engineering/tools-suite/pull/18316/files#L88-L153
Similar to what we already had for isDecimal - https://github.wdf.sap.corp/ux-engineering/tools-suite/blob/0b77f6825ee06e94d1c30328eeb26bf520463e2a/packages/help/core/src/guidedDev/data/guides/addMicroChart/index.ts#L68-L80

Reply:
I see, makes sense then